GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -245,6 +245,7 @@ private void InitializeComponent()
             this.stageFileToolStripMenuItem.Name = "stageFileToolStripMenuItem";
             this.stageFileToolStripMenuItem.Size = new System.Drawing.Size(296, 22);
             this.stageFileToolStripMenuItem.Text = "Stage file(s)";
+            this.stageFileToolStripMenuItem.ShortcutKeyDisplayString = GetShortcutKeyDisplayString(Command.StageSelectedFile);
```

Comment:
Add the following to `ReloadHotkeys` instead:
```C#
            resetFileToParentToolStripMenuItem.ShortcutKeyDisplayString = GetShortcutKeyDisplayString(Command.ResetSelectedFiles);
            stageFileToolStripMenuItem.ShortcutKeyDisplayString = GetShortcutKeyDisplayString(Command.StageSelectedFile);
            unstageFileToolStripMenuItem.ShortcutKeyDisplayString = GetShortcutKeyDisplayString(Command.UnStageSelectedFile);
```

Reply:
This also fixes that the shortcuts were not shown and makes it reasonable to allow reset file by shortcut (still no popup)
![image](https://user-images.githubusercontent.com/6248932/77250315-73d46a80-6c47-11ea-8f0e-9613dd521a63.png)